rust/build.rs

```diff
@@ -5,6 +5,12 @@ mod platform_cfg {
         let ver = glibc_version::get_version().unwrap();
         if ver.major >= 2 && ver.minor >= 28 {
             println!("cargo:rustc-cfg=glibc_renameat2");
+        } else {
+            let more_info = "https://docs.rs/deltalake/0.4.0/deltalake/storage/file/struct.FileStorageBackend.html";
+            println!(
+                "cargo:warning=glibc versions < 2.28 are currently unsupported, glibc version found {}.{}. For more information: {}", 
```

Comment:
for better clarification, glibc <2.28 is supported if users only need to perform read operations. >=2.28 is needed if users need to perform any write using filesystem backend.

Perhaps something along the line of:

```suggestion
                "cargo:warning=glibc version >= 2.28 is required for performing commits to local file system, glibc version found {}.{}. For more information: {}", 
```